define_strong_self.m

```diff
@@ -0,0 +1,10 @@
+// Define Strong Self
+// 
+//
+// IDECodeSnippetCompletionPrefix: vokStrongSelf
+// IDECodeSnippetCompletionScopes: [CodeBlock]
+// IDECodeSnippetIdentifier: 1A45777F-3BA7-409C-BFA2-7B8FBCF20C58
+// IDECodeSnippetLanguage: Xcode.SourceCodeLanguage.Objective-C
+// IDECodeSnippetUserSnippet: 1
+// IDECodeSnippetVersion: 2
+__typeof__(self) strongSelf = weakSelf;
```

Comment:
- why the double underscores around `typeof`?
- shouldn't this use `typeof(weakself)` instead of `typeof(self)`?

I would think it would be: `typeof(weakSelf) strongSelf = weakSelf;`